### PR TITLE
For individual css and js in configs

### PR DESCRIPTION
**Pull request by atomoc**
In the $settings['configuration'] array, you only need to include the files separated by commas into the corresponding elements:
```php
$settings['configuration'] = array(
    'enablePaste' => false,
    'csvseparator' => ',',
    'css' => 'css/my.css',
    'scripts' => 'js/my.js'
);
```

### CODE REVIEW

File: assets/tvs/multitv/includes/multitv.class.php [hunk 1]
```diff
@@ -283,6 +283,8 @@ function prepareSettings($settings)
             $this->configuration['displayLengthMenutext'][] = ($displayLength != -1) ? $displayLength : $this->language['all'];
         }
         $this->configuration['editBoxWidth'] = isset($settings['configuration']['editBoxWidth']) ? $settings['configuration']['editBoxWidth'] : '';
+        $this->configuration['css'] = isset($settings['configuration']['css']) ? $settings['configuration']['css'] : '';
```
Comment (Jako) [inline assets/tvs/multitv/includes/multitv.class.php]: My first idea was a bit overcomplicated:
 
Could you prepare the css array here from the comma separated list?

```
(isset($settings['configuration']['css']) && !empty($settings['configuration']['css'])) ? explode(',', $this->configuration['css']) : array();
```

and remove the tvCSS things in the lines below. After that array_merge in line 838 should be the only thing needed.

Comment (Jako) [inline assets/tvs/multitv/includes/multitv.class.php]: Same with 'js'

Comment (atomoc) [inline assets/tvs/multitv/includes/multitv.class.php]: Ok

**Comment by atomoc** [comment]
Stop. So it does not work. Now I'll figure out why and correct it.

**Comment by atomoc** [comment]
Done

**Comment by Jako** [comment]
Thanks!

**Comment by Jako** [comment]
Now there is only one thing missing before preparing a new version: Updating docs.

**Pull request merged**